parmed/openmm/parameters.py

```diff
@@ -306,12 +306,12 @@ def _write_omm_cmaps(self, dest):
                 dest.write('\n')
             dest.write('  </Map>\n')
         used_torsions = set()
-        for (a1, a2, a3, a4, a5), cmap in iteritems(self.cmap_types):
+        for (a1, a2, a3, a4, a5, a6, a7, a8), cmap in iteritems(self.cmap_types):
             if (a1, a2, a3, a4, a5) in used_torsions: continue
-            used_torsions.add((a1, a2, a3, a4, a5))
-            used_torsions.add((a5, a4, a3, a2, a1))
+            used_torsions.add((a1, a2, a3, a4, a5, a6, a7, a8))
+            used_torsions.add((a8, a7, a6, a5, a4, a3, a2, a1))
             dest.write('   <Torsion map="%d" type1="%s" type=2"%s" type3="%s" '
-                       'type4="%s" type5="%s"/>\n' %
-                       (maps[id(cmap)], a1, a2, a3, a4, a5)
+                       'type4="%s" type5="%s" type6="%s" type7="%s" type8="%s"/>\n' %
```

Comment:
I think OpenMM requires only 5 types (which is the same as requiring `a2==a5`, `a3==a6`, and `a4==a7`).  I'm basing this on [Peter's Drude force field file](https://github.com/pandegroup/openmm/blob/master/wrappers/python/simtk/openmm/app/data/charmm_polar_2013.xml), which has the text:

```XML
...
   <Torsion map="0" class1="CD2O1A" class2="ND2A2" class3="CD31C" class4="CD2O1A" class5="ND2A2"/>
   <Torsion map="0" class1="CD2O1A" class2="ND2A2" class3="CD31C" class4="CD2O1A" class5="ND2A1"/>
   <Torsion map="0" class1="CD2O1A" class2="ND2A2" class3="CD31C" class4="CD2O1A" class5="ND3A3"/>
   <Torsion map="1" class1="CD2O1A" class2="ND2A2" class3="CD32C" class4="CD2O1A" class5="ND2A2"/>
   <Torsion map="1" class1="CD2O1A" class2="ND2A2" class3="CD32C" class4="CD2O1A" class5="ND2A1"/>
   <Torsion map="1" class1="CD2O1A" class2="ND2A2" class3="CD32C" class4="CD2O1A" class5="ND3A3"/>
   <Torsion map="2" class1="CD2O1A" class2="ND3A3" class3="CD31C" class4="CD2O1A" class5="ND2A2"/>
   <Torsion map="2" class1="CD2O1A" class2="ND3A3" class3="CD31C" class4="CD2O1A" class5="ND3A3"/>
 </CMAPTorsionForce>
```

So what you should do is change the for loop to:

```Python
for (a1, a2, a3, a4, _, _, _, a5):
```
and leave the rest of the code as it was.  I think that's the easiest way to do it.